FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -5,6 +5,24 @@
 import Foundation
 
 public final class RemoteFeedLoader: FeedLoader {
+	struct FeedImageStruct: Codable {
+		var imageId: UUID
+		var imageUrl: URL
+		var imageDesc: String?
+		var imageLoc: String?
+
+		enum CodingKeys: String, CodingKey {
+			case imageId = "image_id"
+			case imageUrl = "image_url"
+			case imageDesc = "image_desc"
+			case imageLoc = "image_loc"
+		}
+	}
+
+	struct FeedImageResponse: Codable {
```

Comment:
`FeedImageResponse` is an implementation detail of `RemoteFeedLoader`, and is not supposed to be accessible from outside this class. Please mark it as `private`:

Also, `Codable` is a type alias for the `Encodable` and `Decodable` protocols. In this case, we only need to decode `RemoteFeedImage` from an external representation so `Codable` can be replaced with `Decodable`.
```suggestion
	private struct FeedImageResponse: Decodable {
```